HLTrigger/JSONMonitoring/plugins/L1TriggerJSONMonitoring.cc

```diff
@@ -41,7 +41,7 @@ struct L1TriggerJSONMonitoringData {
     std::vector<unsigned int> l1tAcceptCalibration;  // number of "calibration" events accepted by each L1 trigger
     std::vector<unsigned int> l1tAcceptRandom;       // number of "random" events accepted by each L1 trigger
     std::vector<unsigned int> tcdsAccept;  // number of "physics", "calibration", "random" and other event types
-    int prescaleIndex;                     // prescale column index
+    int prescaleIndex = {};                // prescale column index
```

Comment:
This is supposed to be initialised here, since it is initialised before its first use.

If there is any suspicion that it is used uninitialised, then it should be initialised to `kPrescaleUndefined`, not `0`.
Then, it would make sense to initialise also:
```
stream.processed = 0;
lumisection.prescaleIndex = kPrescaleUndefined;
```

Reply:
kPrescaleUndefined was undefined in that scope (which I blindly tried and now it doesn't compile). would you suggest the change please, I'll commit it :/